### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by artchoi** [kubernetes/minikube#16856]
### What Happened?

stderr:
W0710 07:33:29.527490    4967 initconfiguration.go:119] Usage of CRI endpoints without URL scheme is deprecated and can cause kubelet errors in the future. Automatically prepending scheme "unix" to the "criSocket" with value "/var/run/cri-dockerd.sock". Please update your configuration!
	[WARNING Swap]: swap is enabled; production deployments should disable swap unless testing the NodeSwap feature gate of the kubelet
	[WARNING Service-Kubelet]: kubelet service is not enabled, please run 'systemctl enable kubelet.service'
	[WARNING DirAvailable--var-lib-minikube-etcd]: /var/lib/minikube/etcd is not empty
error execution phase certs/apiserver-kubelet-client: [certs] certificate apiserver-kubelet-client not signed by CA certificate ca: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "minikubeCA")
To see the stack trace of this error execute with --v=5 or higher

### Attach the log file

│    Please run `minikube logs --file=logs.txt` and attach logs.txt to the GitHub issue. 

### Operating System

Ubuntu

### Driver

Docker

**Comment by Boerme84** [comment]
I just had the same issue on my mac. I had a previous minikube running but decided to start from scratch again. So removed container and image and deleted ~/.minikube directory (also did brew uninstall minkube, brew install minikube just to be safe). Got the same error as you did.

So decided to cleanup docker. Before I just removed container and image, but forgot volumes. I just removed all unused volumes, but I guess it could be enough to just remove the minikube volume.
Tried minikube start, and this time it worked.
